be/src/exec/broker_scan_node.cpp

```diff
@@ -433,7 +433,6 @@ void BrokerScanNode::scanner_worker(int start_idx, int length) {
     }
 
     // Update stats
-    _runtime_state->update_num_rows_load_total(counter.num_rows_total);
```

Comment:
If you remove this, rows filtered in this class will not be counted in total loaded rows.

Reply:
Update